src/mlpack/tests/distribution_test.cpp

```diff
@@ -188,6 +189,29 @@ BOOST_AUTO_TEST_CASE(MultiDiscreteDistributionTrainProTest)
   BOOST_REQUIRE_CLOSE(d.Probability("2 1 0"), 0.015625, 1e-5);
 }
 
+/**
+ * Test the LogProbability() function, for multiple points in the multivariate
+ * Discrete case.
+ */
+BOOST_AUTO_TEST_CASE(DiscreteLogProbabilityTest)
+{
+  // Same case as before.
+  DiscreteDistribution d("5 5");
+
+  arma::mat obs("0 2;"
+                "1 2;");
+
+  arma::vec logProb;
+
+
+  d.LogProbability(obs, logProb);
+
+  BOOST_REQUIRE_EQUAL(logProb.n_elem, 2);
+
+  BOOST_REQUIRE_CLOSE(logProb(0), -3.2188758248682, 1e-3);  
+  BOOST_REQUIRE_CLOSE(logProb(1), -3.2188758248682, 1e-3);
+}
+
 /*********************************/
 /** Gaussian Distribution Tests **/
```

Comment:
@rcurtin I think I haven't touched this line. I still don't get why the style error arrises.

Reply:
I can see that line 211 has trailing whitespace.  Not sure why Jenkins labeled the wrong line.  Try removing the whitespace from 211 and I think the test will pass.

Reply:
@rcurtin Please add this PR. I think this will pass all the test. Also, I have changed Discrete Probability implementation and added a new test for discrete probability test.